integration/client_server_test.go

```diff
@@ -238,6 +238,15 @@ func TestClientServer(t *testing.T) {
 			},
 			golden: "testdata/pom.json.golden",
 		},
+		{
+			name: "scan sample.pem with fs command in client/server mode",
+			args: csArgs{
+				Command:          "fs",
+				RemoteAddrOption: "--server",
+				Target:           "testdata/fixtures/fs/secrets/",
+			},
+			golden: "testdata/secrets-rpc.json.golden",
```

Comment:
Can we use the same golden file? We need to make sure the results are the same between standalone mode and client/server mode. For example, we use the same golden files for image scanning.
https://github.com/aquasecurity/trivy/blob/85ca86cb7f97152933e207edabaa8415642b2c4a/integration/client_server_test.go#L49-L55